src/controller/middleware/authentication.js

```diff
@@ -2,8 +2,8 @@ exports.auth = (req, res, next) => {
   if (!req.token) {
     next();
   } else if (req.token && req.token.consultant) {
-    return res.render('consProfile');
+    return res.redirect('/consProfile');
   } else if (req.token && !req.token.consultant) {
-    return res.render('userProfile');
+    return res.redirect('userProfile');
   }
```

Comment:
@AymanAlqoqa Does 'userProfile' need the forward slash too?

Reply:
yes

Reply:
ohhh